OneSignalDevelopers/ngosi
# Styled and centered the Account component and account page

Fixes #23 

## Solution

I added some  tailwind CSS properties that will help center and style the `Account` component  and the `account` page.

## Testing

1. Start Supbase CLI, `supabase start`
2. Start dev server, `yarn dev`
3. Navigate to `localhost:3000`
4. Enter your email and click on the `Send magic link` button
5. Navigate to the http://localhost:54324/monitor page 
6. Note that you will see your email there so click on it 
7. Click on the `Log in` link, this should take you to the `account` page
8. Observe the changes made on the  page

## Resources

###### Before adding styles
![Screen Shot 2022-01-04 at 9 00 00 AM](https://user-images.githubusercontent.com/89279465/148074338-3aac5f21-9ead-4e94-b2f5-154ebd146655.png)

###### After adding styles
![Screen Shot 2022-01-04 at 8 59 33 AM](https://user-images.githubusercontent.com/89279465/148074356-f3bd5033-7c6d-49a5-9233-6bda7c97f805.png)
